### PR TITLE
fix(select): items

### LINKED ISSUE #7190: igx-select long text items
## Description  
igx-select long text items display

 * igniteui-angular versions: 9.0.x, 8.2.x
 * browser: any

## Steps to reproduce  
Open the following [sample](https://stackblitz.com/edit/igx-select-long-text-list-items).

### igniteui-angular 8.2.x:
 * When igx-select item has a long text it's not stretching itself wide enough and the text gets cut off. There isn't any padding on the right side as well.

### igniteui-angular 9.0.x:
 * A horizontal scroll appears when igx-select item has a long text. Again there is no padding on the right side (see the attached image). The selection appears only on the part of the item which is placed below the toggle button. If the text is really long and we scroll further to the right, it seems like there isn't a selected item.

## Expected result  
Since there isn't any description how it should behave in the spec my assumption is: 
 * Either there should be a horizontal scroll, the select should be applied all the way to the end and the padding on the right sight should be similar to this on the left.
 * Or the text should get cut off and the padding on the right side should be again similar to this on the left. 

## Attachments  
* igniteui-angular 8.2.x:
image

* igniteui-angular 9.0.x:
image

### PR DESCRIPTION
**Pull request by desig9stein**
Closes #7190 

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [x] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Comment by ViktorSlavov** [comment]
This PR introduces a behavioral change.
The `igx-drop-down__item` has a display `flex`, but `igx-drop-down__inner` has display `block`, so this might lead to different experience in specific apps that would rely on that.
In particular, this will affect two of our select samples, I'll log an issue in the samples browser for those.

**Comment by simeonoff** [comment]
> Behavioral change caused by missing/moved `flex` display needs to be added to the Changelog

@desig9stein could you update the changelog, please?

**Comment by desig9stein** [comment]
Sorry about the message of the last commit it should be "**updating the changelog.md**"

File: CHANGELOG.md [hunk 1]
```diff
@@ -129,9 +135,6 @@ All notable changes for each version of this project will be documented in this
     ```typescript
     public pinningConfiguration: IPinningConfig = { columns: ColumnPinningPosition.End };
     ```
-    - Added new properties for paging:
```
Comment (Lipata) [inline CHANGELOG.md:132]: unintentionally removed?

**Pull request merged**